tests/Mconfig

```diff
@@ -57,6 +57,22 @@ config TARGET_CLANG_TRIPLE
 	string
 	default "x86_64-linux-gnu"
 
+config KERNEL_CC
+	string
+	default "clang" if ANDROID
+	default "gcc"
```

Comment:
Please remove the `default "gcc"` in example/Mconfig and tests/Mconfig. That way we leave the default compiler to the kernel

Reply:
Removed